packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/SwitchDemo/SwitchDemo.tsx

```diff
@@ -35,4 +64,44 @@ export class SwitchDemo extends React.Component<{}, SwitchState> {
       </React.Fragment>
     );
   }
+
+  renderTable() {
+    const cols = [{ title: 'hello' }];
+    const {
+      table: { data, all }
+    } = this.state;
+    let rr = data;
+    if (!all) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const [first, ...rest] = data;
+      rr = rest;
+    }
+    const rows = rr.map(r => ({
+      cells: [
+        {
+          title: <Switch id={r.id} isChecked={r.isChecked} onChange={this.handleChangeTable(r.id)} />
+        }
+      ]
+    }));
+    return (
+      <div>
+        <Table aria-label="simple table" cells={cols} rows={rows}>
+          <TableHeader />
+          <TableBody />
+        </Table>
+        <Button id="showAll" onClick={this.showAllTable}>
+          Show all
+        </Button>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        {this.renderSimple()}
+        {this.renderTable()}
+      </React.Fragment>
+    );
+  }
```

Comment:
Trying to run the demo app locally, but getting this error.

>Attempted import error: 'EditableTextCell' is not exported from '@patternfly/react-table'.

There are a lot of changes to the demo, considering we're only changing the how the ID is destructured.